src/main/java/me/champeau/gradle/japicmp/ignore/Project.java

```diff
@@ -0,0 +1,24 @@
+package me.champeau.gradle.japicmp.ignore;
+
+import me.champeau.gradle.japicmp.ignore.entity.Entity;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Project {
```

Comment:
Please give a different name to this class. It's easy to misread it as org.gradle.api.Project

Reply:
Done